sbom-generator/scripts/generate-sbom.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+function log() {
+    msg="$1"
+    shift
+    # shellcheck disable=SC2059 # "$1" for printf is assumed safe.
+    printf "$msg\n" "$@" >&2
+}
+
+# Add extra logging if the runner was run with debug logging.
+test -z "${RUNNER_DEBUG+x}" || set -x
+
+# User provided preferences:
+# - CONF_PROJECT_DIR: the path of the project to scan (relative or absolute).
+# - CONF_ECOSYSTEMS: list of ecosystems to scan (e.g., python), whitespace separated.
+# - CONF_RESULT_PATH: the path where to store the SBOM.
+CONF_PROJECT_DIR="${CONF_PROJECT_DIR:-$PWD}"
+CONF_ECOSYSTEMS="${CONF_ECOSYSTEMS:-}"
+CONF_RESULT_PATH=${CONF_RESULT_PATH:-./bom.json}
+
+cmd_args=(
+    "--recurse"
+    "--output=$CONF_RESULT_PATH"
+    "--profile=license-compliance"
+    # Path to the source code to analyze (inside the mounted container)
+    "$CONF_PROJECT_DIR"
```

Comment:
Not true anymore. The shell script used to use docker to run `cdxgen` but it's no longer the case (installs the CLI tool using NPM instead now). The replacement was done due to the container image being around 2 GB thus pulling was slow, and it required a lot of extra logic to share files between the host and the container

```suggestion
    # Path to the source code to analyze
    "$CONF_PROJECT_DIR"
```